Pass language

mturnwall opened the pull request:
You can now pass a language in to override the browser's preferred language.

I also added the example page.

mturnwall committed:
Add ability to override browser's preferred language

mturnwall committed:
Fix misspelling of file name

mturnwall committed:
Fix comments

mturnwall committed:
Create example page

mturnwall committed:
Merge branch 'example-page' into pass-language

mturnwall committed:
Add example page

jgallen23 commented on localization-bar.js:
```diff
@@ -25,6 +36,7 @@ export default class {
    */
   check() {
     this.language = this.findLanguage();
+    console.log(this.language);
```
Oops

jgallen23 commented on example/example.js:
```diff
@@ -0,0 +1,237 @@
+/******/ (function(modules) { // webpackBootstrap
```
We shouldn't check this in

jgallen23 commented on package.json:
```diff
@@ -5,7 +5,9 @@
-    "prepublish": "scriptkit"
+    "prepublish": "scriptkit",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```
Scriptkit already does the building, not sure why we added webpack

jgallen23 committed:
updated to use scriptkit

jgallen23 committed:
updated scriptkit version

jgallen23 committed:
update scriptkit

mturnwall committed:
Remove webpack config and console message

mturnwall commented:
I removed the console log and the webpack config.

Antonio-Laguna committed:
Updating gitignore

Antonio-Laguna committed:
Removing CHANGELOG

Antonio-Laguna committed:
Updating example

Antonio-Laguna committed:
Moving to new flow

Antonio-Laguna committed:
Changes

Using domassist.
Updating/fixing behaviour.
Adding function that can be mocked for testing.

Antonio-Laguna committed:
Adding tests
